src/components/report/VariableControls.js

```diff
@@ -56,11 +61,21 @@ export default {
 		getVariableStyles: function() {
 			let styles = "";
 			let glue = "";
+			let counter = 0;
+			let maxProps = 6;
 			for (const axis of Object.values(this.axes)) {
 				styles += `${glue} "${axis.id}" ${axis.current}`;
 				glue = ",";
+				// Poor man's code formatting
+				if (++counter % maxProps === 0) {
+					glue = `,\n                        `;
+				}
+			}
+			if (styles) {
+				return `font-variation-settings:${styles};`;
```

Comment:
There seems to be quite some duplication between this and `FeatureControls`. Is there an opportunity to extract something?

Reply:
They are indeed quite similar, but have quite a few subtle differences (data structure, CSS, indentation, number of props per line). I _think_ it would make it more complex to have one function to handle them both, than have these separate with some minor duplication.

I don't think these functions will be extended in the future, but if they do, look for opportunities  then?